spec/spec_helper.rb

```diff
@@ -109,3 +109,68 @@ def item_from_foxml(foxml, item_class = Dor::Base, other_class = ActiveFedora::O
   end
   result
 end
+
+################################################################
+# Users
+
+def log_in_as_mock_user(subject)
+  user = double(
+    :webauth_user,
+    :login => 'sunetid',
+    :logged_in? => true
+  )
+  allow(subject).to receive(:webauth).and_return(user)
+end
+
+def admin_user
+  webauth = double(
+    'WebAuth',
+    :login => 'sunetid',
+    :attributes => {'DISPLAYNAME' => 'Admin'}
+  )
+  @admin_user = User.find_or_create_by_webauth(webauth)
+  allow(@admin_user).to receive(:groups).and_return(User::ADMIN_GROUPS)
```

Comment:
Do you need to actually create the user object? (if it just needs to respond to a certain method)